Makefile

```diff
@@ -166,7 +166,12 @@ build_tfhe_full: install_rs_build_toolchain
 .PHONY: build_c_api # Build the C API for boolean and shortint
 build_c_api: install_rs_build_toolchain
```

Comment:
add a dependency on install_rs_check_toolchain

I think we may want to rename the check toolchain to nightly toolchain at some point

Reply:
I added a `install_cbindgen` which depends on `install_rs_check_toolchain`